test/modules/DirectMethodReceiver/src/DirectMethodReceiver.cs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace DirectMethodReceiver
+{
+    using System;
+    using System.Net;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Client;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil.TestResults;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Configuration;
+    using Microsoft.Extensions.Logging;
+
+    class DirectMethodReceiver : IDisposable
+    {
+        Guid batchId;
+        IConfiguration configuration;
+        long directMethodCount = 1;
```

Comment:
start from 0

Reply:
The DM sender has always start at 1. I'm keeping the consistency here.